tests/src/test/scala/cats/tests/ListWrapper.scala

```diff
@@ -44,14 +44,21 @@ object ListWrapper {
 
   def eqv[A : Eq]: Eq[ListWrapper[A]] = Eq[List[A]].on[ListWrapper[A]](_.list)
 
-  val foldable: Foldable[ListWrapper] =
-    new Foldable[ListWrapper] {
-      def foldLeft[A, B](fa: ListWrapper[A], b: B)(f: (B, A) => B): B =
-        Foldable[List].foldLeft(fa.list, b)(f)
+  val traverse: Traverse[ListWrapper] = {
```

Comment:
#1103 is merged - should I resync this or would it tell me if there were conflicts?